src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -0,0 +1,476 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.apache.spark.serializer.KryoRegistrator;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Various types of read anomalies that provide evidence of genomic breakpoints.
+ */
+public class BreakpointEvidence implements Comparable<BreakpointEvidence> {
+    private final short contigIndex;
+    private final short eventWidth;
+    private final int contigStart;
```

Comment:
Is contigStart just the start position of the evidence interval? I find the name a little confusing.